### PR TITLE
[cspp] integrating cspp into decrediton

### PR DESCRIPTION
**Pull request by vctt94**
This PR is a continuation of the integration of cspp at decrediton. #2455 

-  It makes `mixedAccounts` not being able to receive funds (not showing addresses at receive page);
- It makes only possible to send funds from `mixedAccount` at send page; 
- It adds a `isMixTx` method which will help us determine which tx is mixed or not, so we will be able to flag them

    This way we will be able to restore private wallets and guess which account is the mixed account. 

    This code is based on https://github.com/decred/dcrdata/blob/master/txhelpers/cspp.go (thanks @chappjc ).

I think it would be good to add the ability to send funds from other accounts in case the user want to (mark a checkbox, which notifies him about the lack of privacy or the possibility of reducing the anonimous set where these inputs may be spent). 

After this being merged, we will still need to:
 - restore private wallets;
 - Add tutorial for begginers;
 - styling

### CODE REVIEW

File: app/constants/Decred.js [hunk 1]
```diff
@@ -45,3 +45,44 @@ export const MAX_DCR_AMOUNT = 21e14;
...
+//   - 72 bytes DER signature + 1 byte sighash
+//   - OP_DATA_33
+//   - 33 bytes serialized compressed pubkey
+export const	RedeemP2PKHSigScriptSize = 1 + 73 + 1 + 33;
+
+// AtomsPerCoin is the number of atomic units in one coin.
+const AtomsPerCoin = 1e8;
+
+// MaxAmount is the maximum transaction amount allowed in atoms.
+// Decred - Changeme for release
```
Comment (matheusd) [inline app/constants/Decred.js:63]: No need for this comment :P

File: app/helpers/transactions.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        mixedOuts.set(val, 1);
+      }
+    }
+  });
+
+  mixedOuts.forEach( (count, val) => {
+    console.log(val);
+    console.log(count);
+    console.log(mixedOuts);
+    console.log(mixedOuts.get(val));
```
Comment (matheusd) [inline app/helpers/transactions.js]: Extraneaous logs

Comment (matheusd) [reply app/helpers/transactions.js]: Actually I see you've removed on a later commit (was reviewing by commit)

**Comment by alexlyp** [comment]
Needs a rebase, then should be good to go.

**Pull request merged**